src/components/wallet/modal/modal-body.tsx

```diff
@@ -0,0 +1,90 @@
+import { ReactElement } from 'react';
+import * as DialogPrimitive from '@radix-ui/react-dialog';
+import classNames from 'classnames';
+import { getModalMessage, wallets } from './wallets';
+import css from './modal.module.scss';
+import Icon from './icon';
+import CloseIconLight from '@assets/svg/modal/ic-close-light.svg';
+import CloseIconDark from '@assets/svg/modal/ic-close-dark.svg';
+
+export interface DialogContentProps {
```

Comment:
```suggestion
export interface DialogContentProps {
```
Same as the below comments